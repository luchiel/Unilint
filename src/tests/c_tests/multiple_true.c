int main()
{
    int a, b = 0; a = 0; b += a;

    int c; //I am the comment, and comments aren't expressions
    int d = 0, e = 0;
    e -= a;
}