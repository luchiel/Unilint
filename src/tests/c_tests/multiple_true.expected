3:19: two or more expressions at one line
3:26: two or more expressions at one line
